rules/cloud/aws/cloudtrail/aws_cloudtrail_console_login_success_without_mfa.yml

```
@@ -0,0 +1,26 @@
+title: AWS Successful Console Login Without MFA
+id: c134e006-5aef-49dd-9386-62675a251585
```

```suggestion
id: c134e006-5aef-49dd-9386-62675a251585
```
Duplicate to 77caf516-34e5-4df9-b4db-20744fea0a60.

However update 77caf516-34e5-4df9-b4db-20744fea0a60 with ` responseElements.ConsoleLogin: 'Success'`